hydra-tx/test/Hydra/Tx/DepositSpec.hs

```diff
@@ -172,7 +172,7 @@ spec =
           valid `shouldBe` mempty -- All assets in policy must be valid for policy to be valid
           invalid `shouldBe` tokens
         it "splits multiassets correctly" $
-          forAll (genUTxOWithAssetsSized 5) $ \utxo ->
+          forAll (genUTxOWithAssetsSized 5 Nothing) $ \utxo ->
```

Comment:
i know u will hate me for this, but do u think we can replace Nothing with arbitrary?